app/views/layouts/application.html.erb

```diff
@@ -8,9 +8,12 @@
 
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍

Reply:
Okay, I will. once I have it